src/api.js

```diff
@@ -0,0 +1,5 @@
+import axios from "axios";
+
+export default axios.create({
+  baseURL: "http://localhost:8000/api",
+});
```

Comment:
Great use of an axios instance, we can take this one step futher and actually export functions that do the whole api interaction, for example a `getArticles` function that contain the .get, returning a promise containing the data